pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -172,6 +172,8 @@ public static class Instance {
     public static final String CONFIG_OF_PINOT_BROKER_STARTABLE_CLASS = "pinot.broker.startable.class";
     public static final String CONFIG_OF_PINOT_SERVER_STARTABLE_CLASS = "pinot.server.startable.class";
     public static final String CONFIG_OF_PINOT_MINION_STARTABLE_CLASS = "pinot.minion.startable.class";
+    public static final String CONFIG_OF_ALLOW_TABLE_NAME_DOTS = "pinot.table.name.dots.enabled";
```

Comment:
Let's add it under `Helix` similar to `ENABLE_CASE_INSENSITIVE`. Suggest naming it `allow.table.name.dots`